lib/govuk_content_schemas/frontend_schema_generator.rb

```diff
@@ -12,6 +12,7 @@ class GovukContentSchemas::FrontendSchemaGenerator
     redirects
     routes
     update_type
+    change_note
```

Comment:
Is this correct for frontend? Shouldn't change notes be sent as an array in change history or similar?